packages/pluginutils/rollup.config.js

```diff
@@ -4,6 +4,19 @@ import typescript from '@rollup/plugin-typescript';
 
 import pkg from './package.json';
 
+function emitModulePackageFile() {
```

Comment:
let's export this output plugin from https://github.com/rollup/plugins/blob/master/shared/rollup.config.js so we can reuse it